shell/yq.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Wrapper around `yq` that allows use of python-yq or gojq.
+# You can force using gojq by setting the environment variable YQ_USE_GOJQ=true.
+#
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+if [[ -z $GOJQ_VERSION ]]; then
+  GOJQ_VERSION="$(grep ^gojq: "$DIR"/../versions.yaml | awk '{print $2}')"
+fi
+
+if [[ -n $YQ_USE_GOJQ ]]; then
+  use_gojq=true
+elif command -v yq >/dev/null 2>&1; then
+  # Make sure it's the correct yq. The Go yq is not compatible jq syntax.
```

Comment:
This branch of code is going to leave `use_gojq` to an empty value, right? That feels a little weird. Maybe default to `false` or something else?

Reply:
Good catch, will fix.